src/components/actions/ActionListFilters.tsx

```diff
@@ -462,7 +466,7 @@ type ActionListFilterOption = {
 export interface ActionListFilter<Value extends FilterValue = FilterValue> {
   id: string;
   useValueFilterId: string | undefined;
-  filterAction: (value: Value, action: ActionListAction) => boolean;
+  filterAction: (value: Value, action: ActionListAction | IndicatorListIndicator) => boolean;
```

Comment:
Perhaps we could rename these `action` specific properties and variables to be more generic now that this supports indicators too. Later potater though.